<?xml version="1.0"?>
<launch>
    <param name="rigid_mass" value="1" type="double" />

    <group ns="added_mass">
        <param name="x" value="1" type="double" />
        <param name="y" value="1" type="double" />
        <param name="z" value="1" type="double" />
    </group>

    <group ns="rigid_inertia">
        <param name="roll" value="1" type="double" />
        <param name="pitch" value="1" type="double" />
        <param name="yaw" value="1" type="double" />
    </group>

    <group ns="added_inertia">
        <param name="roll" value="1" type="double" />
        <param name="pitch" value="1" type="double" />
        <param name="yaw" value="1" type="double" />
    </group>

    <group ns="drag">
        <group ns="linear">
            <param name="x" value="1" type="double" />
            <param name="y" value="1" type="double" />
            <param name="z" value="1" type="double" />
            <param name="roll" value="1" type="double" />
            <param name="pitch" value="1" type="double" />
            <param name="yaw" value="1" type="double" />
        </group>
        <group ns="quadratic">
            <param name="x" value="1" type="double" />
            <param name="y" value="1" type="double" />
            <param name="z" value="1" type="double" />
            <param name="roll" value="1" type="double" />
            <param name="pitch" value="1" type="double" />
            <param name="yaw" value="1" type="double" />
        </group>
    </group>
</launch>